src/core/reducers/authentication.js

```diff
@@ -1,9 +1,15 @@
+import cookie from 'react-cookie';
+import config from 'config';
+
 export default function authentication(state = {}, action) {
   const { payload, type } = action;
   if (type === 'SET_JWT') {
-    return { token: payload.token };
+    return { ...state, token: payload.token };
   } else if (type === 'SET_CURRENT_USER') {
     return { ...state, username: payload.username };
+  } else if (type === 'LOG_OUT_USER') {
+    cookie.remove(config.get('cookieName'));
```

Comment:
I'm going to file an issue to look at removing the JWT from the HTML. In the mean time I actually think this shouldn't be in the reducer so I'm going to move it to `mapDispatchToProps` in `<App />`.